docs/content/en/blog/news/pipecd-best-practice-01.md

```diff
@@ -0,0 +1,38 @@
+
+At a glance, PipeCD is composed of 2 components, the Control Plane and the Piped(s). As the figure above, depending on your task/role, you need to work with different components of PipeCD.
+
+1. As a operator - platform team, most of your time working with PipeCD is to operate the PipeCD Control Plane. You may also need to cooperate with the product team, helping them install Piped(s) to their applications running cluster depending on your company team's structure. The space surrounded by a <span style="color: green;">green border</span> shows the operators working area.
+2. As a developer - product team, most of your time working with PipeCD is to control your applications' development via the PipeCD application configuration files. You may also need to cooperate with the platform team to install Piped(s) to your applications running cluster if necessary. The space surrounded by a <span style="color: blue;">blue border</span> shows the developers working area.
+
+### Tips to operate PipeCD cluster
+
+1. Only a single Control Plane needs to be installed and will be operated by the operators (platform team). The installed Control Plane doesn't have to be in the same cluster as your applications running cluster or the Piped installed cluster. Just make sure the Piped(s) can connect to the Control Plane via outbound requests is enough.
+
+2. Developers from the product team may need to self-install piped into their applications running cluster (due to some credential-related issues, if the platform team does not manage those credentials).\
+\
+When using PipeCD, developers only need to care about [application configuration](/docs/user-guide/adding-an-application/) - which defines the pipeline so that piped(s) can use them to deploy your applications. Interaction with the applications is mainly done via the PipeCD web console and the configuration files stored on Git.
+
+> As an __operator__, PipeCD [control-plane](/docs/operator-manual/control-plane/) and [piped](/docs/operator-manual/piped/)(s) are what you should care about. As a __developer__, you should care about [piped](/docs/operator-manual/piped/) which installed in your applications running cluster and your [applications' PipeCD configurations](/docs/user-guide/adding-an-application/).
+
+3. It is possible to use a single piped to manage deployments of all the applications to the clusters other than the one in which piped is being installed. However, due to credentials and security issues, __piped should be installed independently in each applications running cluster, and only manages the deployments of applications on that cluster__. In case you want to deploy applications that are required to be installed across clusters/regions/environments, and want to follow this rule of thumb, please refer to the [deployment-chain](/docs/user-guide/deployment-chain/) feature.
```

Comment:
> However, due to credentials and security issues, __piped should be installed independently in each applications running cluster, and only manages the deployments of applications on that cluster__

However, as security best practices and avoid generating cluster credentials, we highly recommend running each Piped inside each cluster and that Piped will only manage the applications on that cluster.